htdocs/install/page_modcheck.php

```diff
@@ -63,37 +64,51 @@ function xoDiagIfWritable( $path) {
 	return xoDiag( $error ? -1 : 1, $error ? 'Not writable' : 'Writable' );
 }
 
+function imCheckRequirements()
+{
+	$requirement['server_api']['description']=PHP_SAPI;
+	$requirement['server_api']['result']=php_sapi_name();
+	$requirement['server_api']['status']=true;
+
+	$requirement['php_version']['description']=_PHP_VERSION;
+	if (version_compare( phpversion(), '5.6', '>=')) {
+		$requirement['php_version']['status']=1;
+	} else {
+		$requirement['php_version']['status']=0;
+	}
+	$requirement['php_version']['result']=phpversion();
+
+	$requirement['mysql']['description']="MySQL Handler";
+	$requirement['mysql']['result']=in_array("mysql",PDO::getAvailableDrivers(),TRUE) ? ? SUCCESS : FAILED;
+	$requirement['mysql']['status']=in_array("mysql",PDO::getAvailableDrivers(),TRUE) ? true : false;
```

Comment:
There seems to be an extra '?' in line 82

Reply:
copy-paste error, fixed now.